modules/ROOT/pages/design-publish.adoc

```diff
@@ -1,59 +1,63 @@
 
-For example, suppose that the file example.raml is among the files in a project. If the root file of the project does not reference the file with any `!include` tag, then the root file doesn't reference the file directly. Also, if no file, in a chain of references from the root file, itself references example.raml, then example.raml is not referenced indirectly from the root file. Therefore, example.raml is unreferenced and should probably not be among the files in your project.
+For example, if the file `example.raml` is among the files in a project but the root file of the project does not reference `example.raml` with an `!include` tag, the root file doesn't reference `example.raml` directly. If none of the files in the chain of references from the root file directly references `example.raml`, it is not referenced indirectly from the root file. Therefore, `example.raml` is unreferenced and should probably not be among the files in the project.
 
+If you do not remove them before you begin, unreferenced files are detected when you select the option to publish to exchange and you are given options on whether or not to publish them. 
 
-== Procedure
+[publish-api-specs]
+== Publish an API Specification
 
-. Open the API-specification project that you want to publish, if it is not already open.
-. In the top-right corner of API Designer, click *Publish* and then click *Publish to Exchange*.
+. In API Designer, open the API-specification project that you want to publish.
+. Click *Publish* and then click *Publish to Exchange*.
 +
-*Result*: This dialog is opened.
+image:apid-publish-dialog-api-specs.png[The publish dialog shown when you publish from API Designer to Exchange]
 +
-image:apid-publish-dialog-api-specs.png[]
+<1> *Asset version*: The version number to be displayed in the *Version* section of the page that Anypoint Exchange creates for the project. You can republish assets that are in the *In Development* lifecycle state using the same asset version number. The asset version for assets that are in the *Stable* lifecycle state must be incremented to the next valid version when republished. For a description of this field, see <<asset-version>>.
+<2> *API version*: The version number of the API specification. For a description of this field, see <<api-version>>.
+<3> *LifeCycle State*: The lifecycle state of the asset. Assets can be published from API Designer as *In Development* or *Stable*. Assets cannot be deprecated from API Designer. 
+<4> *Business group ID*: The ID of the business unit in Anypoint Platform to which the project belongs. 
+<5> *Asset name*: The name that Anypoint Exchange uses to identify the published project. You should typically keep the value as is.
+<6> *Asset ID*: The ID that Anypoint Exchange uses to identify the published project. You should typically keep the value as is.
```

Comment:
Would it be correct to replace "the value as is" with "the current [default?] value"? 
I ask only because "as is" strikes me as idiomatic of "as it is."

Reply:
Changed along with this line of thinking (default) and have sent a note to Eng to get more information the use cases where someone might want to change it.